pkgs/by-name/si/signal-export/package.nix

```diff
@@ -25,6 +26,8 @@ python3.pkgs.buildPythonApplication rec {
     emoji
     markdown
     pysqlcipher3
```

Comment:
I believe this is the wrong package: The unmaintained https://github.com/rigglemania/pysqlcipher3/.

The package wants "sqlcipher3-wheels", which lives at https://github.com/laggykiller/sqlcipher3 according to pypi and is itself a fork of https://github.com/coleifer/sqlcipher3 and seems to mainly add a workflow to build wheels and do some openssl-shenanigans. 

I suggest we try to package https://github.com/coleifer/sqlcipher3 directly and patch signal-export to use "sqlcipher3" instead of "sqlcipher3-wheels". I can do that but it might take me a few days.